### PR TITLE
add moola

### PR DESCRIPTION
**Pull request by minrk**
cc @funsim, @sebastkm

### CODE REVIEW

**Comment by minrk** [comment]
includes a couple of patches applied to master since release 0.1.6 to fix the dependencies.

File: recipes/moola/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - moola
+
+about:
+  home: https://github.com/funsim/moola
+  license: LGPL 3
+  license_family: LGPL
+  # FIXME: license file is missing from sdist in 0.1.6
+  # this is fixed in master
+  # license_file: LICENSE
```
Comment (jakirkham) [inline recipes/moola/meta.yaml]: xref: https://github.com/funsim/moola/pull/9

File: recipes/moola/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b5936cef0d2a29432784dea9e4813e3653d5277a66bac67bf5d89ba912ec7ce
+  patches:
+    - imports.patch
```
Comment (jakirkham) [inline recipes/moola/meta.yaml]: Was this submitted upstream? Guessing PR ( https://github.com/funsim/moola/pull/8 ). Could we add a comment above this line citing that?

**Comment by jakirkham** [comment]
One comment. Otherwise LGTM.

File: recipes/moola/meta.yaml (hunk 2 above)
Comment (minrk) [reply recipes/moola/meta.yaml]: added note that all patches are merged upstream.

**Comment by minrk** [comment]
@jakirkham thanks! Yes, these patches are all already merged upstream. Added a note that they should be removed on the next update.

**Comment by minrk** [comment]
ping @conda-forge/staged-recipes this should be set, I think.

**Comment by scopatz** [comment]
@funsim, @sebastkm are you OK with being maintainers here?

**Comment by sebastkm** [comment]
@scopatz Yes, and @funsim is also ok with it.

**Comment by minrk** [comment]
Thanks!

**Pull request merged**